webpack.config.extensions.js

```diff
@@ -106,7 +106,7 @@ function getWebpackConfig() {
 			rules: [
 				TranspileConfig.loader( {
 					workerCount,
-					configFile: path.join( __dirname, 'babel.config.extensions.js' ),
+					configFile: path.join( __dirname, 'babel.config.js' ),
```

Comment:
Will omitting this line default to this file anyway? I like the clarity of having it here tho.

Reply:
I _think_ omission would result in the same behavior. I'm not too worried about having this extremely polished because I'm anticipating major babel changes before the next Jetpack release.

Reply:
Valid point. 👍